packages/prettier-config-edu/index.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  arrowParens: 'always',
+  bracketSpacing: false,
+  plugins: [require('prettier-plugin-tailwindcss')],
+  singleQuote: true,
+  trailingComma: 'all',
+};
```

Comment:
In a future PR we'll actually start using these prettier and eslint configs to format/lint this repo! Didn't want to do that, yet, though, to keep the diff smaller.